test/junit/scala/collection/IteratorTest.scala

```diff
@@ -213,6 +213,16 @@ class IteratorTest {
     assertEquals(it.drop(1).next(), it.drop(1).drop(1).next())
   }
 
+  @Test def dropNegativeReturnsSelf():Unit= {
+    val iterator = List(1).iterator
+    assertEquals(iterator.drop(Int.MinValue), iterator)
+  }
+
+  @Test def dropMaxValueReturnsEmpty() :Unit = {
+    val iterator = List(1).iterator
+    assertEquals(iterator.drop(Int.MaxValue), Iterator.empty)
```

Comment:
`assertFalse(it.hasNext)`

Reply:
Thanks, as your prevous review comment, I think I need to change it to equal.